src/Utils/cartUtil.js

```diff
@@ -0,0 +1,8 @@
+const decreaseQtyHandler = (cartDispatch,product)=>{
+    if(product.qty <= 1){
+        cartDispatch({type:'REMOVE_FROM_CART',payload:product})
+    }else{
+        cartDispatch({type:'DECREASE_QTY',payload:product})
```

Comment:
Suggestion, you can make an object for the action.type constants and add 'REMOVE_FROM_CART' and 'DECREASE_QTY' as a property to that object.

Use Case: Your code editor will provide suggestion for the actions helping you code faster

Reply:
Hey, Om thank you for the suggestion will refactor it later.